### ISSUE TITLE
Circuit Breaker

### ISSUE THREAD

**Issue by MCTyler** [BluSunrize/ImmersiveEngineering#1645]
Sneaky suspicion this is you crashing the client...I have the latest build built from your repository.....
<details>
Time: 10/28/16 5:57 AM
Description: Tesselating block model

java.lang.NullPointerException: Tesselating block model
    at blusunrize.immersiveengineering.client.ClientUtils.convertConnectionFromBlockstate(ClientUtils.java:1305)
    at blusunrize.immersiveengineering.client.models.smart.ConnModelReal$AssembledBakedModel.func_188616_a(ConnModelReal.java:131)
    at blusunrize.immersiveengineering.client.models.smart.ConnModelReal.func_188616_a(ConnModelReal.java:69)
    at net.minecraft.client.renderer.BlockModelRenderer.func_187497_c(BlockModelRenderer.java:147)
    at net.minecraftforge.client.model.pipeline.ForgeBlockModelRenderer.func_187497_c(ForgeBlockModelRenderer.java:86)
    at net.minecraft.client.renderer.BlockModelRenderer.func_187493_a(BlockModelRenderer.java:72)
    at net.minecraft.client.renderer.BlockModelRenderer.func_178267_a(BlockModelRenderer.java:52)
    at net.minecraft.client.renderer.BlockRendererDispatcher.func_175018_a(BlockRendererDispatcher.java:81)
    at net.minecraft.client.renderer.chunk.RenderChunk.func_178581_b(RenderChunk.java:278)
    at net.minecraft.client.renderer.chunk.ChunkRenderWorker.func_178474_a(ChunkRenderWorker.java:119)
    at net.minecraft.client.renderer.chunk.ChunkRenderWorker.run(ChunkRenderWorker.java:47)
    at java.lang.Thread.run(Thread.java:745)
</details>
I was operating the Circuit breaker when it happened......Err, check that...I had the latest build when you said this was fixed...I'll try build 48....

**Comment by BluSunrize** [comment]
Sneaky suspicion: You didn't read the contribution guidelines =P
Crashlogs go into spoilers (pastebin, gist, etc.) if you'd please.

Also @malte0811 I have zero proof, but I'll totally blame you! <3

**Comment by MCTyler** [comment]
Sry :D

**Comment by MCTyler** [comment]
Although Git doesn't show this...Use the details html script for spoilers.....

**Comment by MCTyler** [comment]
Build 48 seems to be good....

**Comment by malte0811** [comment]
I think that this is still a bug in build 48, a race condition between networking/integrated server and rendering (Yes, my fault). The problem about race conditions is that they are hard to reproduce, but I will try to fix this today or tomorrow (I have some half-done stuff for NBTEdit I want do finish first).
On a second thought, this might be real easy to fix, I just have to remember why I used the connection vertices field instead of the method when I wrote the wire models.
